src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -220,7 +253,6 @@ public void WhenNewIssuesAreFound_OutOfRangeIssuesAreIgnored_ListenersAreUpdated
             SetupIssuesFilter(out var _);
 
             // Sanity check
-            testSubject.LastIssues.Should().BeNull();
```

Comment:
I've removed the `LastIssues` property from the interface and the class - not needed.